LocalSettings.php

```diff
@@ -727,6 +727,7 @@
 			'Religion' => 'religion',
 			'Science' => 'science',
 			'Software/Computing' => 'software',
+			'Song Contest' => 'sc',
 			'Sports' => 'sport',
```

Comment:
Why use just `sc`? Why not songcontest?

Reply:
To save time. There are two other examples above.

Reply:
This is not as descriptive though and that's because those include special characters that I don't think can be in this.

Reply:
Done.

Reply:
Thanks!